framework/auth/campaigns.py

```diff
@@ -24,7 +24,7 @@ def get_campaigns():
             # Native campaigns: PREREG and ERPC
             newest_campaigns = {
                 'prereg': {
-                    'system_tag': 'prereg_challenge_campaign',
+                    'system_tag': 'osf_prereg_campaign',
```

Comment:
What determined the new system_tag value?

Reply:
Me! No actually you have a good point. I was just trying to remove the word `challenge` because it's not a challenge anymore. We could continue to use it internally without ill effect, but I thought lets make the end of the prereg_challenge_campaign very obvious.